### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by Albinpirela**
to make corrections before going to master 🐌

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,12 +6,193 @@
...
+        <header>
+            <h1>Stand up for Somalia</h1>
+            <hr>
+            <img src="christine.png">
+            <h2>Christine Kolesnikov</h2>
+            <p>Hypnosis - Senior Editor</p>
+            <button><a href=""><i class="fa-brands fa-facebook"></i> Shere</a></button>
+            <button><a href=""><i class="fa-brands fa-twitter"></i> Tweet</a></button>
+            <button><a href=""><i class="fa-brands fa-linkedin"></i> Share</a></button>
+            <button><a href=""><i class="fa-sharp fa-solid fa-link"></i> Copy</a></button>
```
Comment (hendridg) [inline index.html]: Hi @Albinpirela , great job so far, please if you have any list of components in your project you need to put a list.
```
<ul>
   .
   .
   .
   .
</ul>
```

File: index.html [hunk 2]
```diff
@@ -6,12 +6,193 @@
...
+        <article>
+            <img src="people1.png">
+            <h2>People are dying of hunger...
+                there’s no water’</h2>
+            <p>In Somalia, more than 6 million people are in need of food assistance more than half the population.
+                But even inside the camps, food is scarce.</p>
+            <hr>
+            <img src="let's.png">
+            <h2>Let’s help them. I know you can</h2>
+            <p>Girls, boys and families are experiencing food insecurity, higher food and other commodity prices coupled
```
Comment (hendridg) [inline index.html]: this is great, eny article in the project need the project tag :+1:

**Review by hendridg — CHANGES_REQUESTED** [review]
# :recycle: Change Required :recycle: 

Hi @Albinpirela , you can see the difference between this project and another you developed before, with code quality, you are almost there! 

My suggestion is please change the name of PR, this name doesn't say anything about the PR, another stuff you miss is the git workflow, I don't see any feature branch `feature/****`

File: index.html (hunk 2 above)
Comment (Albinpirela) [reply index.html]: Thank you for your revision @hendridg I made several branches which I merged to develop there are 5 functions I made 5 pull requests with different branches all of them were merged to deelop, I will take your advice into account to make the corresponding revisions, thanks 👍

File: index.html (hunk 1 above)
Comment (Albinpirela) [reply index.html:31]: gracias por la sugerencia @hendridg voy a hacer esa corrección de inmediato 📑

Comment (Albinpirela) [reply index.html]: thanks for the suggestion @hendridg I'll make that correction right away 📑

**Review by hendridg — APPROVED** [review]
# :green_circle: Approved :green_circle: 

Great job @Albinpirela , the only recommendation I can say is in the `ul` list, which is not necessarily the `button`  element but is not a big problem,  so is time to merge!

**Comment by Albinpirela** [comment]
I will continue studying to improve myself. I sincerely appreciate your attention 😃

**Pull request merged**